#Machine Configuration
x_vel_limit_mps:                      0.5
y_vel_limit_mps:                      0.5
accel_limit_mps2:                     1.0
decel_limit_mps2:                     1.0
dtz_decel_limit_mps2:                 1.0
yaw_rate_limit_rps:                   1.0
yaw_accel_limit_rps2:                 1.0
wheel_diameter_m:                     0.1524
wheel_base_length_m:                  0.50173
wheel_track_width_m:                  0.42835
gear_ratio:                           5.0
motion_while_charging:                0
motion_ctl_input_filter:              0

#Control Configuration
p_gain_rps_per_rps:                   1.8
i_gain_rps_per_rad:                   1.58
d_gain_rps_per_rps2:                  0.0
fdfwd_gain_rps_per_motor_rps:         1.0
p_error_limit_rps:                    33.0
i_error_limit_rad:                    20.0
d_error_limit_rps2:                   0.0
i_error_drain_rate_rad_per_frame:     0.0001
output_limit_rps:                     12566.37                     
input_target_limit_rps:               12566.37

